src/server/transaction.cc

```diff
@@ -7,6 +7,7 @@
 #include <absl/strings/match.h>
 
 #include "base/logging.h"
+#include "glog/logging.h"
```

Comment:
why do you need this?

Reply:
it's clangd 🙄 Will remove it

Reply:
how the fuck did you succeed to use clangd? It does not work for me due to confusion with stdlib++ implementations

Reply:
@romange I use clangd as well? What are you using? Isn't this the goto language server?